games-action/openclonk-7.0.recipe

```diff
@@ -0,0 +1,127 @@
+SUMMARY="A free multiplayer action game where you control clonks"
+DESCRIPTION="OpenClonk is a free multiplayer action game in which you control \
+clonks, small, but witty and nimble humanoid beings. The game is mainly about \
+mining, settling and fast-paced melees."
+HOMEPAGE="http://www.openclonk.org/"
+COPYRIGHT="2009-2016 The OpenClonk Team and contributors
+	2006-2007 Alexander Post
+	2004-2013  Armin Burgmeier
+	2005-2006, 2008-2010  Asmageddon
+	2010, 2012  Benjamin Herr
+	2006, 2012  Bernhard Bonigl
+	2009-2010  Carl-Philip Hänsch
+	2001  Carlo Teubner
+	2009, 2011-2013  David Dormagen
+	2011-2012  Felix Wagner
+	2006  Florian Groß
+	2004-2012  Günther Brammer
+	2006-2008  Julian Raschke
+	2010-2012  Julius Michaelis
+	2010-2011  Maikel de Vries
+	2010  Manuel Riecke
+	2009  Mark Haßelbusch
+	2009-2012  Martin Plicht
+	1998-2009  Matthes Bender
+	2001  Michael Käser
+	2009  mizipzor
+	2009-2013  Nicolas Hake
+	2013  Oliver Schneider
+	2010  Peewee
+	2001-2011  Peter Wortmann
+	2012  Philipp Kern
+	2009-2010  Richard Gerum
+	2001-2013  Sven Eberhardt
+	2004-2005, 2009-2013  Tobias Zwick"
+LICENSE="ISC"
```

Comment:
ISC isn't default in the licenses for Haiku, so you need to include this in this pull request, aside from that it's not looking to bad :+1:

Reply:
How do I do that?

Reply:
Why didn't it show the same error before?

Reply:
Could be that you have another package installed that also includes this license, you can see an example here: https://github.com/haikuports/haikuports/tree/75a3cf5384f650242f4b8a1094ab305d33ed99e7/app-shells/mksh

Reply:
Do I need to create a new pull request for that?

Reply:
No, what you need to do is to create a folder named `licenses` at the same folder with your recipe, and then put your license there

Reply:
should that be done in this pull request?

Reply:
Yes, it should be included in this PR with your recipe from the begining
Just do what @fbrosson said

Reply:
Yes :)

Reply:
Should this pull request be closed, and opened again with the new file?

Reply:
Link to the ISC license is provided in the COPYING file ...

Reply:
@vrindag, you can add the license easily.
Oops, you need to create a subdirectory for the recipe. 
For your convenience, here's what you can copy/paste:
```
cd /boot/home/haikuports/games-action
mkdir openclonk
git mv openclonk-7.0.recipe openclonk
git commit -m "move recipe to its own directory" openclonk-7.0.recipe openclonk/openclonk-7.0.recipe
cd openclonk
mkdir licenses
cp ../../media-libs/libass/licenses/ISC licenses
git add licenses/ISC
git commit -m "add license" licenses/ISC
```

Reply:
@vrindag, I just noticed that your branch is called "patch-2".
So I guess you are editing your recipe locally and then copying your changes to GitHub.
That's OK for us, but you should really consider using git on your local branch. It's much faster than having to edit a file on your local system and another on GitHub's website.

If your patch-2 branch is only on GitHub, here's what you can do to grab it:

```
cd /boot/home/haikuports

git remote remove origin
git remote add origin https://github.com/vrindag/haikuports.git
git remote add upstream https://github.com/haikuports/haikuports.git

git pull upstream master

cd games-action
if [ -f openclonk-7.0.recipe ]; then mv openclonk-7.0.recipe openclonk-7.0.recipe.bak; fi

git checkout -b patch-2 master
git fetch vrindag patch-2
git merge --no-edit vrindag/patch-2
git rebase master

git mv openclonk-7.0.recipe openclonk
git commit -m "move recipe to its own directory" openclonk-7.0.recipe openclonk/openclonk-7.0.recipe
cd openclonk
mkdir licenses
cp ../../media-libs/libass/licenses/ISC licenses
git add licenses/ISC
git commit -m "add license" licenses/ISC

git push --force --set-upstream vrindag patch-2
```

Reply:
> Should this pull request be closed, and opened again with the new file?

No, you don't need to close this PR. Moreover, it's not recommended to close a PR just to open another one. There is nothing you can't do in this PR. The only reason why you would need to open another PR would be after you delete your remote branch (accidentally, of course) or if you delete your repo (also accientally).

What you can do, however, is squash all your commits (on your local branch) and push the resulting commit to your remote "patch-2" branch, i.e. here, on GitHub's server. This will allow you to get rid of all the small commits. But this is not required.

Reply:
thanks